### PR TITLE
Scanner refactorings

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1241)

### CODE REVIEW

File: scanner/src/funTest/kotlin/scanners/ScanPathTest.kt [hunk 1]
```diff
@@ -24,8 +24,7 @@ import com.here.ort.utils.safeDeleteRecursively
 import com.here.ort.utils.test.ExpensiveTag
```
Comment (fviernau) [inline scanner/src/funTest/kotlin/scanners/ScanPathTest.kt]: -use plural form for function: "make some previously protected properties / function public,"
-is 'load' intended? : "with unknown download load time"
-consider removing '.' and / or ',' from the end of the listing items.

File: scanner/src/main/kotlin/LocalScanner.kt [hunk 2]
```diff
@@ -307,7 +307,7 @@ abstract class LocalScanner(config: ScannerConfiguration) : Scanner(config), Com
 
         log.info { "Running $this version ${scannerDetails.version} on path '${path.absolutePath}'." }
```
[the remark this answers is not in the document]
Comment (sschuberth) [reply scanner/src/main/kotlin/LocalScanner.kt]: Done.

File: scanner/src/funTest/kotlin/scanners/ScanPathTest.kt (hunk 1 above)
Comment (sschuberth) [reply scanner/src/funTest/kotlin/scanners/ScanPathTest.kt]: Actually, it should have been singular for "properties", fixed.

Comment (sschuberth) [reply scanner/src/funTest/kotlin/scanners/ScanPathTest.kt]: Done.

File: scanner/src/funTest/kotlin/scanners/ScanPathTest.kt [hunk 3]
```diff
@@ -24,46 +24,68 @@ import com.here.ort.utils.safeDeleteRecursively
...
-import io.kotlintest.TestResult
+import io.kotlintest.Spec
 import io.kotlintest.shouldBe
 import io.kotlintest.specs.StringSpec
 
 import java.io.File
 
 class ScanPathTest : StringSpec() {
     private val config = ScannerConfiguration()
+    private val path = File("../LICENSE")
```
[the remark this answers is not in the document]
Comment (sschuberth) [reply scanner/src/funTest/kotlin/scanners/ScanPathTest.kt]: Done.

**Pull request merged**